src/pkg/packager/lint/lint.go

```diff
@@ -25,120 +22,109 @@ import (
 	"github.com/xeipuuv/gojsonschema"
 )
 
-// ZarfSchema is exported so main.go can embed the schema file
-var ZarfSchema embed.FS
-
-func getSchemaFile() ([]byte, error) {
-	return ZarfSchema.ReadFile("zarf.schema.json")
+// FileLoader is an interface for reading files, it decouples the lint package from the go embed package
+type FileLoader interface {
+	ReadFile(path string) ([]byte, error)
 }
```

Comment:
Seems like a simple enough change, but I am curious why defining our own `FileLoader` interface is preferable over using the standard library `embed` package?

Reply:
You cannot use simulate go embed in test AFAIK so this interface makes it possible to unit test the validate function

Reply:
Would you mind adding that as a comment for context?